src/index.js

```diff
@@ -26,7 +26,13 @@ export default (config) => {
       },
       gtagOptions: {},
     },
-    modules: ['tracking', 'youtube', 'facebook', 'google'],
+    modules: ['tracking', 'youtube', 'facebook', 'google', 'twitter', 'vimeo'],
```

Comment:
@Molochem We shouldn't change the default list of modules; that's a breaking change.

In the readme, let's be clear about which modules are enabled by default and which ones are options that can be added if desired.